opencensus-shim/src/main/java/io/opentelemetry/opencensusshim/OpenTelemetryB3FormatImpl.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.opencensusshim;
+
+import static io.opentelemetry.opencensusshim.SpanConverter.mapSpanContext;
+
+import io.opencensus.trace.SpanContext;
+import io.opencensus.trace.propagation.TextFormat;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.propagation.TextMapPropagator;
+import io.opentelemetry.extension.trace.propagation.B3Propagator;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nullable;
+
+public class OpenTelemetryB3FormatImpl extends TextFormat {
```

Comment:
```suggestion
class OpenTelemetryB3FormatImpl extends TextFormat {
```

Reply:
Also I think this should be `OpenTelemetryTextFormat` and it can have a constructor to accept any `TextMapPropagator` instead of being hardcoded to b3. Presumably we want to add w3c, or basically any propagator a user wants to configure.